Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -78,11 +79,7 @@ - (instancetype)initWithCollectionView:(UICollectionView *)collectionView
 
 - (NSMutableArray *)collapsedForms
 {
-    if (_collapsedForms) return _collapsedForms;
-
-    _collapsedForms = [NSMutableArray new];
-
-    return _collapsedForms;
+    return _collapsedForms = (_collapsedForms) ?: [NSMutableArray new];
```

Comment:
Hm, what is the result of ?: if _collapsedForms is not nil? Will it be _collapsedForms (then it's OK) or nil? Maybe it would make the code more readable if you defined a value between ? and :.